README.md

```diff
@@ -2510,16 +2510,30 @@ Produces a random number between the inclusive lower and upper bounds. If only o
     _.random(1.2, 5.2);
     // => a floating-point number between 1.2 and 5.2
 
-    //Native
-    const random = (lower, upper) => {
-      if(!upper || typeof upper === 'boolean') {
-        upper = lower;
-        lower = 0;
+    //Native ES6
+    const random = (...args) => {
```

Comment:
can you use default params here so that the `if` statements below can be cleaned up?

Reply:
It's not easy I think. Depending on the different number of arguments, the order will be different
I have no idea how to do it right and clean

Reply:
Why not using an object in this case? I think it's fine to have a slightly different signature than lodash.

Reply:
Well, what's the point? I may write a function with all required arguments :) 
`random(min, max, float)`
And then the code will be nice and shiny!

Reply:
@stevemao  I think this solution is OK

Reply:
Although the solution is great. I think the point here is if you have similar signature as lodash, might as well using `lodash.random` instead of reimplementing it in Native JS.

If the random function has to work exactly same as lodash,
```
_.random(1,0)  // return 1 or 0,
random(1, 0) // always return 1
```

So, is it important to inherit all the lodash characteristic?